internal/pkg/earpointer/analysis.go

```diff
@@ -65,33 +67,61 @@ func run(pass *analysis.Pass) (interface{}, error) {
 // Insert into the state all the local references and global references.
 func (vis *visitor) initReferences(ssainput *buildssa.SSA) {
 	state := vis.state
+	prog := ssainput.Pkg.Prog
 	for _, member := range ssainput.Pkg.Members {
-		if m, ok := member.(*ssa.Global); ok {
+		switch m := member.(type) {
```

Comment:
We're now covering every `Member` type except `*ssa.NamedConst`.  It could be good to make it clear that wasn't an oversight, either with an explicit "do nothing" case or just a comment here at the switch.

Not necessary if it's coming in a near-future PR.